src/Controller/CaseReminderController.php

```diff
@@ -64,7 +64,7 @@ public function new(CaseEntity $case, Request $request): Response
             $this->entityManager->persist($reminder);
             $this->entityManager->flush();
 
-            return $this->redirectToRoute('case_index');
+            return $this->redirect($request->headers->get('referer'));
```

Comment:
Are you absolutely sure that `$request->headers->get('referer')` is set?

Reply:
Replaced with 
```
$redirectUrl = $request->headers->get('referer') ?? $this->generateUrl('case_index');

return $this->redirect($redirectUrl);
```